playwright/e2e/crypto/utils.ts

```diff
@@ -221,6 +221,9 @@ export async function logIntoElement(page: Page, credentials: Credentials, secur
         await page.locator(".mx_AuthPage").getByRole("button", { name: "Verify with Recovery Key" }).click();
 
         const useSecurityKey = page.locator(".mx_Dialog").getByRole("button", { name: "use your Recovery Key" });
+        // If the user has set a security passphrase, they'll be prompted for that first and have to click
```

Comment:
is "security passphrase" referring to what I now call a recovery passphrase? Or a recovery key? (Or both?)

Reply:
Yes, fixed and also hopefully made clearer.